src/ruuvitag-adapter.ts

```diff
@@ -157,6 +155,20 @@ export class RuuviTagAdapter extends Adapter {
     this.knownDevices = {};
     addonManager.addAdapter(this);
 
+    // @tim - the manifest that is loaded (I can't work out how) does not have the id
+    //        The id is required to load the configuration, so I hack it in here
+    //        Sometimes I've seen a generated manifest.json in the lib directory
+    manifest.id = manifest.id || 'ruuvitag-adapter';
+
+    const config = getDefaultConfig();
+    const db = new Database(manifest.id);
+    db.open()
+    .then(() => { return db.loadConfig(); })
+    .then((c) => {
+      mergeLoadedConfig(config,c);
+    })
+    .catch((e) => console.error(e));
```

Comment:
There is an even simple way to do this.
```suggestion
    mergeLoadedConfig(config, manifest.config);
```